_posts/2017-09-29-make-connections.html

```diff
@@ -0,0 +1,46 @@
+<figure>
+	<img src="/assets/uploads/2017/09/twine.png" alt="" /></a>
+	<figcaption>Conversational user journey mapped with <a href="https://twinery.org">Twine</a></figcaption>
+</figure>
+<p>Whatever the terminology, it implies one core factor: users. Developers need to design new journeys and plan conversation flows, within the limitations imposed by the current APIs and tools. We have been thinking about how and when our readers might want to use voice interactions: at home, at work, in the morning, on the go... Voice devices create new modes of accessibility, for example, visually impaired users could switch from screen readers to a voice device for a more fluid interaction (e.g not saying 'Click this button', but 'Read me the article'). But they also brig new constraints: a page cannot be refreshed at the click of a button, or a file downloaded with a right click anymore. When designing we have to be mindful of breaking people's screen habits.</p>
+<h3>"Alexa, tell me about &lt;CompanyName&gt;"</h3>
+<p>We have explored the possibilities to develop for two major players: Amazon and Google. At the time of trying out the Echo Dot, development was still limited and a very manual task of filling in data. We were also limited by our own source of data for companies, not having it readily available for consumption. We decided to switch to Google, which was slightly more flexible with dynamic data.</p>
+
+<h2>"OK Google"</h2>
+<p>FT Labs has a longer term goal where we want to allow rich conversations to take place between our audience and voice devices, this game is a step along the path to that capability.</p>
+<p>To develop Make Connections, we used <a href="https://api.ai/">API.ai</a> and its Action on Google integration. The game uses the correlations service, based on our previous exploration of the <a href="https://labs.ft.com/2015/07/six-degrees-of-angela-merkel/">6 degrees of Angela Merkel</a>.</p>
+<p>In the same spirit, players much choose from 3 propositions which person has been mentioned in the same FT article as the one in the question. The game only looks at recent articles (published within a week), so the answers will be different over time.</p>
+<blockquote>
+	<p>"Theresa May was mentioned in an article with which one of the following people?"<p>
+	<ol>
+		<li>Martin Gruenberg</li>
+		<li>Richard Cousins</li>
+		<li>Mark Carney</li>
+	</ol>
+	<p><cite>Correct answer on 02/10/2017: Mark Carney</cite></p>
+</blockquote>
+<p>An answer can be made by saying the option number or the person's name. Upon answering, the player is offered more context on the connection: the title of the article where the correlation occured. If the answer was correct, another question comes up and the score increases until a wrong answer is given, or no more connections are found.</p>
```

Comment:
or no more connections *can be presented as questions*.

Reply:
*Every person chosen, either as the focus for a question or as the correct answer to a question, is removed from the pool of candidates for constructing subsequent questions.*

Reply:
Since the most-frequently cited people are chosen for questions initially, the game gets progressively harder as more and more obscure people appear in questions.